docs/static/security/tls.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[ls-http-ssl]]
+=== Configure TLS Encryption
```

Comment:
Like Logstash (or any other client connecting to Elasticsearch), you need to generate security certificates for that service. For example, when you [configure Metricbeat to use TLS](https://elasticdocstest.netlify.app/playground/esStackSecurityOverview#configure-metricbeat-to-use-tls), we're including the certificate generation and Metricbeat configuration in the same steps to keep users from bouncing around the docs.

I think that integrating the Logstash TLS steps into that larger security context makes sense, because you have to configure TLS in Elasticsearch and Kibana before doing so on other clients.